python/deltalake/__init__.py

```diff
@@ -36,3 +57,18 @@ def to_pyarrow_dataset(self) -> pyarrow.dataset.Dataset:
 
     def to_pyarrow_table(self) -> pyarrow.Table:
         return self.to_pyarrow_dataset().to_table()
+
+    def to_pyarrow_schema(self) -> pyarrow.Schema:
```

Comment:
since we are not converting a delta table to  pyarrow schema here, i think `pyarrow_schema` would have been a better method name.

Reply:
That's right, I will change the method_name to `pyarrow_schema`